### PR TITLE
renamed "requiresNoCD" to "requiresPatch"

### LINKED ISSUE #860: rename requiresNoCD
software + scripts
e.g. field in `ScriptDTO` and the `BooleanProperty` in `ApplicationFilter`

see #856

### PR DESCRIPTION
**Pull request by plata**
fixes #860

requires PlayOnLinux/Scripts#252

### CODE REVIEW

**Comment by qparis** [comment]
Can you merge the master to this branch?

**Comment by plata** [comment]
I don't understand why the test is failing.

File: phoenicis-repository/src/main/java/org/phoenicis/repository/dto/ScriptDTO.java [hunk 1]
```diff
@@ -97,13 +97,13 @@ public boolean equals(Object o) {
...
                 .append(testingOperatingSystems, scriptDTO.testingOperatingSystems).append(free, scriptDTO.free)
-                .append(requiresNoCD, scriptDTO.requiresNoCD).append(script, scriptDTO.script).isEquals();
+                .append(requiresPatch, scriptDTO.requiresPatch).append(script, scriptDTO.script).isEquals();
     }
 
     @Override
     public int hashCode() {
         return new HashCodeBuilder(17, 37).append(scriptName).append(scriptSource).append(compatibleOperatingSystems)
-                .append(testingOperatingSystems).append(free).append(requiresNoCD).append(script).toHashCode();
+                .append(testingOperatingSystems).append(free).append(requiresPatch).append(script).toHashCode();
```
Comment (qparis) [inline phoenicis-repository/src/main/java/org/phoenicis/repository/dto/ScriptDTO.java]: Same formatting issue here...

**Comment by qparis** [comment]
Have you tried to manually run the tests from your IDE?

**Comment by plata** [comment]
Yes. The checksum comparison for `testDeserialize()` fails but I don't understand the reason.

**Comment by qparis** [comment]
Try to put back the "is" keyword behind requiresPatch

**Comment by plata** [comment]
Thanks. That solved it.

**Pull request merged**